doctr/file_utils.py

```diff
@@ -76,6 +75,18 @@
         " is installed and that either USE_TF or USE_TORCH is enabled."
     )
 
+if _torch_available:
```

Comment:
@Fabioomega We can remove this.

2 options:

We pin the lower boundary to >= 2.0.0 here https://github.com/mindee/doctr/blob/9045dcfc9c5c837b06fcda8e802f7cf1d95bd18c/pyproject.toml#L61
https://github.com/mindee/doctr/blob/9045dcfc9c5c837b06fcda8e802f7cf1d95bd18c/pyproject.toml#L105
and `torchvision>=0.15.0`

or we mention in the docs that this requires >= 2.0.0 for compile and >=2.4.0 for compile + fullgraph

@odulcy-mindee wdyt ? 
We are already at 2.4.0 so i would prefer the >=2.0.0 pin